Code/Editor/CryEdit.cpp

```diff
@@ -323,7 +323,7 @@ CCryEditDoc* CCryDocManager::OpenDocumentFile(const char* lpszFileName, bool bAd
         }
     }
 
-    if (pOpenDocument != nullptr)
+    if (!reopenIfSame && pOpenDocument != nullptr)
```

Comment:
what is going to happen with the current IO handle if the doc is already opened (`pOpenDocument != nullptr`) in the case that `reopenIfsame` is true?
Is it releasing the previous IO handle? It could be dangerous to keep an IO handle opened.
should it close it first before trying to open it again?

Reply:
Its not a open handle or anything like. If you follow the code you can see that the variable is just what GetEditor()->GetOpenDocument() returns. So that same pointer will be pointing to the new document after loading.